### DISCUSSION TITLE
[General] Can we make there be fewer submission boxes for this forum?

### DISCUSSION THREAD

**Discussion by PatrickAlphaC** [zkSync-Community-Hub/zksync-developers#506] [General] [answered]
### Environment

Testnet

### Provide a brief description of the functionality you're trying to implement.

Like maybe just one or two

### What is the specific issue or error you're encountering?

Not sure we need this box

### Can you share the error messages or logs you're receiving, if any?

I mean, I guess this is a good reminder, but probably could be in the default template for the first box. 

### Have you made any recent changes to the contract before encountering this issue?

not needed for sure

### Are there any external libraries or contracts that your contract interacts with?

This is probably a good reminder for the main box

### Can you provide the relevant portions of your contract code where the issue is occurring?

no

### Have you tried to isolate the problem, and if so, what were the results?

not needed

### What steps have you already taken to try to resolve the issue?

not needed

**Comment by dutterbutter** [comment]
Absolutely. We just had a discussion on how these templates need to be re-worked. 

Of course any suggestions here is welcomed. 

Cc: @bxpana @uF4No

**Reply by bxpana** [reply, marked as answer]
@PatrickAlphaC updated the general one, how does it look?

**Comment by PatrickAlphaC** [comment]
much better :)
